scalableminds/webknossos
# Unified annotation restore

This PR changes the version restore to use one single tab / a joined version restore for all layers

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Open an annotation
- trace around :racing_car: incl. volume annotation
- open the version restore and preview some versions
- The volume actions should have their volume layer's name in the action description
- try out restoring an older version

### Issues:
- contributes to https://github.com/scalableminds/webknossos/pull/7917#issuecomment-2413488436
